recipe/meta.yaml

```diff
@@ -8,11 +8,11 @@ source:
   url: http://download.osgeo.org/gdal/{{ version }}/gdal-{{ version }}.tar.xz
   sha256: d32223ddf145aafbbaec5ccfa5dbc164147fb3348a3413057f9b1600bb5b3890
   patches:
-    # disable 12 bit jpeg on Windows as we aren't using internal jpeg
+    # disable 12 bit libjpeg-turbo  on Windows as we aren't using internal libjpeg-turbo
     - disable_jpeg12.patch  # [win]
```

Comment:
Is this patch of concern?

Reply:
> Is this patch of concern?

I believe the patch is still appropriate for this branch (the automated change in the comment though isn't appropriate)